src/hotspot/cpu/aarch64/aarch64_neon.ad

```diff
@@ -5289,6 +5289,7 @@ instruct vsll8B_imm(vecD dst, vecD src, immI shift) %{
   predicate(n->as_Vector()->length() == 4 ||
             n->as_Vector()->length() == 8);
   match(Set dst (LShiftVB src (LShiftCntV shift)));
+  match(Set dst (LShiftVB src shift));
```

Comment:
Does it make sense if changing the mid-end codes to make sure the immediate shift is broadcasted to a vector?

Reply:
This could avoid fixing the missing rules for all related architecture.

Reply:
`x86` misses the rule `match(Set dst (LShiftVB src (LShiftCntV imm))); `. If we choose to fix it by adding IR `LShiftCntV`.  Anyway,  adding a `LShiftCntV` here may be more clear. I will refact the code.